docs/guides/universal-profile/transfer-lyx.md

```diff
@@ -42,31 +42,17 @@ For a regular LYX transfer, the parameters will be:
 - `_value`: the amount of LYX we want to transfer (in Wei).
 - `_data`: empty (`0x` since we are just transferring LYX).
 
-### Interacting via the Key Manager
+### Permissions checking through the Key Manager
 
-Most of the functions on the UP contract, such as [`setData(...)`](../../standards/smart-contracts/erc725-contract.md#setdata---erc725y) and [`execute(...)`](../../standards/smart-contracts/erc725-contract.md#execute---erc725x) can only be called by the [`owner`](../../standards/smart-contracts/erc725-contract.md#owner). Therefore if we want to use our UP to do meaningful things, **all interactions should go through the KM**.
+Most of the functions on the UP contract, such as [`setData(...)`](../../standards/smart-contracts/erc725-contract.md#setdata---erc725y) and [`execute(...)`](../../standards/smart-contracts/erc725-contract.md#execute---erc725x) can only be called by the [`owner`](../../standards/smart-contracts/erc725-contract.md#owner) or by the controller addresses that are allowed in the owner contract of the Universal Profile, thanks to **LSP20**. In our case it is the Key Manager the owner o0f the Universal Profile and the contract that verifies the permissions of the controller addresses, therefore if we want to use our UP to do meaningful things, we should set proper [**permissions**](../../standards/universal-profile/lsp6-key-manager.md#permissions) for the address that we want to allow to use our Universal Profile.
```

Comment:
This whole section and the figure are now irrelevant.

We can just explain that to transfer LYX, we need either:
- the permission transfer value + the address we are sending LYX to in the list Allowed Calls.
- the permission SUPER_TRANSFERVALUE

The figure can be deleted.

<img width="936" alt="image" src="https://user-images.githubusercontent.com/31145285/232045516-294bb6bc-3dfd-4c5d-8a02-2246680e5343.png">